scripts/memory-scorecard.sh

```
@@ -0,0 +1,335 @@
+  work="$(jq -r '.work' <<<"$row")"; expect="$(jq -r '.expect' <<<"$row")"
+  forbid="$(jq -r '.forbid // ""' <<<"$row")"; stale="$(jq -r '.stale_token // ""' <<<"$row")"
+  realistic="$(jq -r '.realistic_claude_md // ""' <<<"$row")"
+  steelman="$(jq -r '.steelman_claude_md // ""' <<<"$row")"
+  facts="$(jq -c '.plant // []' <<<"$row")"
+  local run=1
+  while [ "$run" -le "$RUNS" ]; do
+    local base="$WORKROOT/$id-r$run"
+    echo "-- $dim/$id (run $run/$RUNS) [$plant_mode]"
+
+    # memory-on
+    local mb="$base/on" db="$mb/memory.db" ns="rb-sc-$id-r$run"
```

Shellcheck (0.11.0)

[warning] This assignment is used again in this 'local', but won't have taken effect. Use two 'local's.

(SC2318)